server/src/main/resources/application.conf

```diff
@@ -6,6 +6,7 @@ application.cdn = ${?APPLICATION_CDN}
 
 parsers.text.maxLength = 100m
 
+portcode = "xxx"
```

Comment:
Is this deliberate?

Reply:
The tests won't run without specifying a port code now. Not entirely sure why - possibly because the tests refer to config now and our config depends on a portcode having been specified to form the app's url

Reply:
The issue's not in our code but rather in the application.conf where the url is constructed: play.http.context = "/v2/"${portcode}"/live"

I think the config in the tests is actually because I've left the API query for Brian as a test

Reply:
👍